src/test/scala/bazeltools/bsp4bazel/LspTestProcess.scala

```diff
@@ -207,16 +211,22 @@ case class LspTestProcess(workspaceRoot: Path):
       client: BspClient,
       actions: List[Lsp.Action],
       exitSwitch: Deferred[IO, Either[Throwable, Unit]]
-  ): IO[List[InitializeBuildResult | Unit]] =
+  ): IO[List[InitializeBuildResult | WorkspaceBuildTargetsResult | Unit]] =
```

Comment:
I wonder if we should have a `type` defined here for this union especially if we keep adding to it.